casper/src/main/scala/coop/rchain/casper/api/BlockAPI.scala

```diff
@@ -351,6 +351,8 @@ object BlockAPI {
     case Some(Processing) =>
       DeployServiceResponse(success = false,
                             "No action taken since other thread is already processing the block.")
-    case None => DeployServiceResponse(success = false, "No block was created.")
+    case None =>
+      DeployServiceResponse(success = false,
+                            "No block was created. The rnode might be in read-only mode.")
```

Comment:
Maybe say "Either no new deploys have been received or the node may be in read-only mode".

Maybe we should also give a warning in the deploy response too about the node being read only...

Reply:
Done

The rest will be for https://rchain.atlassian.net/browse/RHOL-663